### PR TITLE
Jared Edit & Update Event

### PR DESCRIPTION
**Pull request by Jared-Waldroff**
Edit & update event are done. When a host is logged in a button will show up under the event giving them the option to edit the event. Once they click it, it will take them to a page only accessible by that button. All the info will be displayed in the text boxes ready to be updated. Once updated the event becomes unverified until the admin approves it so you won't see the event be on the front page until then.

### CODE REVIEW

File: PassIO/templates/editevent.html [hunk 1]
```diff
@@ -22,7 +22,8 @@
   {% include 'navbar.html' %}
```
Comment (ayhchen13) [inline PassIO/templates/editevent.html:22]: editevent.html looks great - easy to use with current field values displayed in each box and user just need to change the current value to update!

File: PassIO/app.py [hunk 2]
```diff
@@ -60,8 +60,32 @@ def eventapproval():
     return render_template('eventapproval.html',user=CurrentUser, unapproved_events=unapproved_events)
 @app.route('/editevent')
 def editevent():
-    event = mongo.db.Event.find({"verified": "verified"})
-    return render_template('editevent.html',user=CurrentUser, event=event)
+    event_id = request.args.get('id')  # Get event ID from query parameter
```
Comment (ayhchen13) [inline PassIO/app.py:63]: I think we can integrate this with what I have done in the "Event history" page for hosts, where it display all the events hosted by the specific logged in host:
Screenshot 2024-04-09 at 9 19 56 PM
, cuz I think currently hosts can edit events not hosted by them.

Let's merge to main and ill make the changes after.

**Review by ayhchen13 — APPROVED** [review]

**Review by AngryMagnets — APPROVED** [review]
Looks good

**Comment by Jared-Waldroff** [comment]
Added ability to insert number of tickets and the date of the event

**Pull request merged**